### PR TITLE
Change 'normal' termination reason

### PR DESCRIPTION
**Pull request by vkatsuba**
Changed 'normal' termination reason to expected 'cp_inactivity_timeout'

### CODE REVIEW

**Comment by ebengt** [comment]
The failed Gitlab in CI could be what I experienced last week. Random failure with
Cannot connect to the Docker daemon at tcp://localhost:2375. Is the docker daemon running?
Solved by adding version to docker image. Example:
    image: docker:19.03.8
    services:
        - docker:19.03.8-dind

File: apps/ergw_core/src/gtp_context.erl [hunk 1]
```diff
@@ -612,7 +612,7 @@ handle_event(info, {'DOWN', _MonitorRef, Type, Pid, _Info}, State,
     {next_state, shutdown, Data};
 
 handle_event({timeout, context_idle}, stop_session, State, Data) ->
-    delete_context(undefined, normal, State, Data);
+    delete_context(undefined, cp_inactivity_timeout, State, Data);
```
Comment (ebengt) [inline apps/ergw_core/src/gtp_context.erl:615]: Who wants/expects
cp_inactivity_timeout
It is not part of ergw.

**Review by ebengt — APPROVED** [review]

**Comment by vkatsuba** [comment]
> The failed Gitlab in CI could be what I experienced last week. Random failure with
> 
> Cannot connect to the Docker daemon at tcp://localhost:2375. Is the docker daemon running?
> 
> Solved by adding version to docker image. Example:
> 
>     image: docker:19.03.8
> 
>     services:
> 
>         - docker:19.03.8-dind
> 
> 

Thanks but as I know that the GitLab CI should be migrated to GitHub actions soon, but maybe this fix also will be need in GH actions.

File: apps/ergw_core/src/gtp_context.erl (hunk 1 above)
Comment (vkatsuba) [reply apps/ergw_core/src/gtp_context.erl:615]: You are right it is not a part of erGW - this is a part of erGW-AAA and will used for termination cause mapping.

**Pull request merged**